IM_calculation/scripts/calculate_ims.py

```diff
@@ -10,12 +10,27 @@
 
 import argparse
 import os
+import logging
+from pathlib import Path
 
 import IM_calculation.IM.im_calculation as calc
 from IM_calculation.Advanced_IM import advanced_IM_factory
 from qcore import constants
 from qcore import utils
-from qcore import qclogging
+
+
+if __name__ == "__main__":
+    from mpi4py import MPI
+    from qcore import MPIFileHandler
+
+    comm = MPI.COMM_WORLD
+    rank = comm.Get_rank()
+    size = comm.Get_size()
+    master = 0
+    is_master = not rank
+
+    logger = logging.getLogger("IM_calc_rank_%i" % comm.rank)
+    logger.setLevel(logging.DEBUG)
```

Comment:
I don't think calculate_ims gets imported anywhere, so this shouldn't need a name main wrapper
Doesn't hurt though